api/account/info.go

```diff
@@ -24,12 +24,14 @@ import (
 
 type InfoResponse struct {
 	Username        string `json:"username"`
+	DiscordId       string `json:"discordId"`
+	GoogleId        string `json:"googleId"`
 	LastSessionSlot int    `json:"lastSessionSlot"`
 }
 
 // /account/info - get account info
-func Info(username string, uuid []byte) (InfoResponse, error) {
-	response := InfoResponse{Username: username, LastSessionSlot: -1}
+func Info(username string, discordId string, googleId string, uuid []byte) (InfoResponse, error) {
+	response := InfoResponse{Username: username, LastSessionSlot: -1, DiscordId: discordId, GoogleId: googleId}
```

Comment:
at this point you should format this differently. Having it all in one line isnt readable for that much parameters